samcli/lib/build/workflow_config.py

```diff
@@ -251,17 +255,18 @@ def get_workflow_config(runtime, code_dir, project_dir, specified_workflow=None)
             identifiers=[specified_workflow, runtime],
             specified_workflow=specified_workflow,
         )
+        assert selector
```

Comment:
Is this a common practice to use assert inside a production code?

Reply:
absolutely not, good catch!